sdk/react/style/controlBarStyles.json

```diff
@@ -32,7 +32,7 @@
     "fontFamily": "AvenirNext-DemiBold",
   },
   "completeTimeStyle": {
-    "flex":0,
+    "flex":1,
```

Comment:
Sure.

On Tuesday, April 5, 2016, lental <notifications@github.com> wrote:

> In sdk/react/style/controlBarStyles.json
> <https://github.com/ooyala/native-skin/pull/404#discussion_r58636901>:
>
> > @@ -32,7 +32,7 @@
> >      "fontFamily": "AvenirNext-DemiBold",
> >    },
> >    "completeTimeStyle": {
> > -    "flex":0,
> > +    "flex":1,
>
> not sure how I feel about this - can you revert?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/native-skin/pull/404/files/0bdc26156f353eab876062552645fa31de4480cd#r58636901>
>


-- 
Thanks,

*Unnath Kumar Woona*
Android Developer - Playback team
ukumar@ooyala.com | (469 353-3239

<http://www.ooyala.com/> <http://www.facebook.com/ooyala>
<http://www.linkedin.com/company/ooyala> <http://www.twitter.com/ooyala>